### PR TITLE
Add ActionRadio

### PR DESCRIPTION
**Pull request by skjnldsv**
Bug:
- [x] event is not fired on the unchecked radio.

### CODE REVIEW

**Comment by skjnldsv** [comment]
Bump :)

File: src/components/ActionRadio/ActionRadio.vue [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+			if (this.$refs.radio.checked) {
+				/**
+				 * Emitted when the radio is checked
+				 * @type {Event}
+				 */
+				this.$emit('check', true)
+
+				// if part of a set, firing events on all the other
+				if (this.name) {
```
Comment (ChristophWurst) [inline src/components/ActionRadio/ActionRadio.vue]:
```suggestion
				if (!this.name) {
```

and early return to lower the number of indentations

File: src/components/ActionRadio/ActionRadio.vue [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+					const parentAction = GetParent(this, 'Actions')
+					if (parentAction) {
```
Comment (ChristophWurst) [inline src/components/ActionRadio/ActionRadio.vue]:
```suggestion
					if (!parentAction) {
```

and early return

File: src/components/ActionRadio/ActionRadio.vue [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+			 * @type {Event}
+			 */
+			this.$emit('change', event)
+
+			if (this.$refs.radio.checked) {
+				/**
+				 * Emitted when the radio is checked
+				 * @type {Event}
+				 */
+				this.$emit('check', true)
```
Comment (ChristophWurst) [inline src/components/ActionRadio/ActionRadio.vue]: even if $emit is async I would argue that it should be fired *after* the `update:checked` was done, so we're 200% sure the listeners listening to change will get the new value when they read it

File: src/utils/GetChildren.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+ */
+
+/**
+ * Get the all the children matching the provided name
+ *
+ * @param {Object} context the context to search from (usualy this)
+ * @param {string} name the children name
+ * @returns {Object[]} the children components
+ */
+const GetChildren = function(context, name) {
```
Comment (ChristophWurst) [inline src/utils/GetChildren.js]:
```suggestion
const getChildren = function(context, name) {
```

this is not c#

File: src/utils/GetParent.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+ */
+
+/**
+ * Get the first parent component matching the provided name
+ *
+ * @param {Object} context the context to search from (usualy this)
+ * @param {string} name the parent name
+ * @returns {Object} the parent component
+ */
+const GetParent = function(context, name) {
```
Comment (ChristophWurst) [inline src/utils/GetParent.js]:
```suggestion
const getParent = function(context, name) {
```

**Pull request merged**